project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
+                              :exclusions [org.clojure/clojure]]]
```

Comment:
I'd propose consolidating the `:ezbake` and `:pe` versions of this dep to the top level. Would there be a good reason to keep them separate @kbarber @shrug @waynr?